contentElements/MultiLineInputElement.js

```diff
@@ -70,7 +77,12 @@ MultiLineInputElement.prototype.init = function (variableName) {
  * This function is used recursively to retrieve an array with all modifiers.
  * @param {Array} modifiersArr - this is an array that holds all modifiers.
  */
+// MultiLineInputElement.prototype.getAllModifiers = function (modifiersArr) {
```

Comment:
please clean up unnecessary commented out code.

Reply:
This comment actually was initially in the file, should I remove it still?

Reply:
The unnecessary comment was removed